Description: Used canned example for SpecialFanoFourfolds documentation
 Otherwise, we run out of memory when building on i386.
Bug: https://github.com/Macaulay2/M2/issues/1539
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-10-27

--- a/M2/Macaulay2/packages/SpecialFanoFourfolds.m2
+++ b/M2/Macaulay2/packages/SpecialFanoFourfolds.m2
@@ -2408,7 +2408,61 @@
 Outputs => {FunctionClosure => {"which takes the ideal of a (general) point ", TEX///$p\in\mathbb{P}^5$///, " and returns the unique rational curve of degree ", TEX///$e$///, ", ", TEX///$(3e-1)$///, "-secant to ", TEX///$S$///, ", and passing through ", TEX///$p$///, " (an error is thrown if such a curve does not exist or is not unique)"}}, 
 EXAMPLE {"-- A general cubic fourfold of discriminant 26"|newline|"X = specialCubicFourfold(\"Farkas-Verra C26\",ZZ/33331);", "describe X", "time f = detectCongruence X;", "p = point ring X -- random point on P^5", "time C = f p -- 5-secant conic to the surface", "assert(codim C == 4 and degree C == 2 and codim(C+(first ideals X)) == 5 and degree(C+(first ideals X)) == 5 and isSubset(C, p))"}, 
 PARA{"The same method can be also applied to a ", TO SpecialGushelMukaiFourfold, ". In this case it will detect and return a congruence of (2e-1)-secant curves of degree e inside the unique del Pezzo fivefold containing the GM fourfold."}, 
-EXAMPLE{"-- A general GM fourfold of discriminant 20"|newline|"X = specialGushelMukaiFourfold(\"surface of degree 9 and genus 2\",ZZ/33331);", "describe X", "time f = detectCongruence X;", "Y = source map X; -- del Pezzo fivefold containing X", "p = point Y -- random point on Y", "time C = f p -- 3-secant conic to the surface", "S = sub(first ideals X,Y);", "assert(dim C -1 == 1 and degree C == 2 and dim(C+S)-1 == 0 and degree(C+S) == 3 and isSubset(C, p))"}, 
+EXAMPLE{PRE(///i7 : -- A general GM fourfold of discriminant 20
+     X = specialGushelMukaiFourfold("surface of degree 9 and genus 2",ZZ/33331);
+/// | ///
+o7 : SpecialGushelMukaiFourfold (Gushel-Mukai fourfold containing a surface of degree 9 and sectional genus 2)
+/// | ///
+i8 : describe X
+/// | ///
+o8 = Special Gushel-Mukai fourfold of discriminant 20
+     containing a surface in PP^8 of degree 9 and sectional genus 2
+     cut out by 19 hypersurfaces of degree 2
+     and with class in G(1,4) given by 6*s_(3,1)+3*s_(2,2)
+     Type: ordinary
+     (case 17 of Table 1 in arXiv:2002.07026)
+/// | ///
+i9 : time f = detectCongruence X;
+S: surface of degree 9 and sectional genus 2 in PP^8 cut out by 19 hypersurfaces of degree 2
+phi: quadratic rational map from 5-dimensional subvariety of PP^8 to PP^13
+Z=phi(del Pezzo fivefold)
+number lines containing in Z and passing through the point phi(p): 7
+number 1-secant lines to S passing through p: 6
+number 3-secant conics to S passing through p: 1
+     -- used 19.3376 seconds
+/// | ///
+i10 : Y = source map X; -- del Pezzo fivefold containing X
+/// | ///
+i11 : p = point Y -- random point on Y
+/// | ///
+o11 = ideal (t  + 14118t , t  + 3234t , t  - 16296t , t  - 5674t , t  -
+              7         8   6        8   5         8   4        8   3
+      -----------------------------------------------------------------------
+      12127t , t  - 1329t , t  + 3304t , t  + 779t )
+            8   2        8   1        8   0       8
+/// | ///
+o11 : Ideal of Y
+/// | ///
+i12 : time C = f p -- 3-secant conic to the surface
+     -- used 0.347885 seconds
+/// | ///
+o12 = ideal (t  - 1000t  + 8254t  - 12393t , t  + 10116t  + 7449t  - 15895t ,
+              5        6        7         8   4         6        7         8
+      -----------------------------------------------------------------------
+      t  + 10858t  + 13401t  + 13664t , t  - 11215t  + 13587t  - 5150t , t  -
+       3         6         7         8   2         6         7        8   1
+      -----------------------------------------------------------------------
+      1898t  + 4900t  + 14451t , t  - 7830t  + 1802t  - 14129t )
+           6        7         8   0        6        7         8
+/// | ///
+o12 : Ideal of Y
+/// | ///
+i13 : S = sub(first ideals X,Y);
+/// | ///
+o13 : Ideal of Y
+/// | ///
+i14 : assert(dim C -1 == 1 and degree C == 2 and dim(C+S)-1 == 0 and degree(C+S) == 3 and isSubset(C, p))
+///)},
 SeeAlso => {coneOfLines}} 
 
 document {Key => {SpecialCubicFourfold}, 
